conf/evolutions/default/9.sql

```diff
@@ -0,0 +1,12 @@
+
+# --- !Ups
+
+ALTER TABLE collections ADD COLUMN content_prefill_window_start TIMESTAMPTZ;
+ALTER TABLE collections ADD COLUMN content_prefill_window_end TIMESTAMPTZ;
+UPDATE  collections  SET content_prefill_window_start = CONCAT(cast((select issue_date from edition_issues where edition_issues.id = (select issue_id from fronts where fronts.id = front_id)) as text), ' 01:00:00.00 UTC')::TIMESTAMP where prefill IS NOT NULL;
```

Comment:
Why from 1am?

Reply:
i will add a comment on that, 
I think issue_date is in London time
that one is in UTC and effectively i was getting day before with 23:00:00 time
like issue_date was `2019-10-11` and then the start and end date as part fo migration were `2019-10-10T23:00:00:00`

So it is to keep the same day

Reply:
done